styles/main.css

```diff
@@ -72,8 +72,7 @@ header nav {
 
 .bio {
     text-indent: 3em;
-    grid-row: 1 / 4; 
-    grid-column: 1 / 4;
+    grid-column: 1 / span 2;

```

Comment:
You don't need to specify which row .bio starts and ends on here.  You're only interested in making sure it spans 2 columns, for which the grid-column declaration you have here is enough.